### ISSUE TITLE
Objective C doesn't see label delegate property and protocol definition

### ISSUE THREAD

**Issue by LugonjaAleksandar** [apploft/ExpandableLabel#67]
There are @objc declarations missing for delegate property and for protocol declaration in the library, and therefore Objective-C doesn't see this property at all and can't be used within Obj-C project. Please include this fix in your next release (add @objc  before the delegate property and before protocol). Would be a good idea to create sample project in Objective-C just to be sure everything written is Swift is accessible from objective-c.

**Comment by tachtevrenidis** [comment]
@LugonjaAleksandar take a look and let me know

**Comment by LugonjaAleksandar** [comment]
@tachtevrenidis Yes, that is exactly what was missing. Thanks for this fast fix. :)

**Comment by tachtevrenidis** [comment]
alright let's see if we can get this merged! @tinora
